src/utility/HistoryStore.js

```diff
@@ -33,10 +33,10 @@ export default class HistoryStore {
 
   save() {
     const items = this.items;
-    let value = JSON.stringify({[this.key]: items});
-    while (items.length >= 0 && !this.storage.set(this.key, value)) {
+    let value = JSON.stringify({ [this.key]: items });
+    while (items.length > 10 && !this.storage.set(this.key, value)) {
```

Comment:
I set it to 10 so it would match the max history length constant. If it's just set to 0 it just won't add anything to the history